dashboard/impl/src/java/org/sakaiproject/dash/dao/impl/DashboardDaoImpl.java

```diff
@@ -846,6 +862,26 @@ public boolean deleteTaskLocks(String task) {
            return new ArrayList<AvailabilityCheck>();
 		}
 	}
+	public boolean isScheduleAvailabilityCheckMade(AvailabilityCheck availabilityCheck){
```

Comment:
select.AvailabilityChecks.entry only takes one param: entityRef. I think only this param is needed here for isScheduleAvailabilityCheckMade()